plots/README.md

```diff
@@ -14,17 +14,13 @@ You need to build lighthouse first.
 ### Generating & viewing charts
 
 ```
-# View all commands
-$ cd plots
-$ yarn run
-
 # Run lighthouse to collect metrics data
-$ yarn measure
+$ node measure.js
```

Comment:
you got a whole CLI for measure.js now!. at the very least you should paste the `--help` output into the readme